homeassistant/components/nuki/__init__.py

```diff
@@ -53,6 +54,16 @@ async def async_setup_entry(hass, entry):
 
     hass.data.setdefault(DOMAIN, {})
 
+    # Migration of entry id
+    if isinstance(entry.unique_id, int):
+        new_id = parse_id(entry.unique_id)
+        if entry.title == entry.unique_id:
+            hass.config_entries.async_update_entry(
```

Comment:
Collect the parameters in a dict and pass the params dict as keyword arguments in a single call to `async_update_entry` instead of writing two different calls.
```py
params = {"unique_id": new_id}
if entry.title == entry.unique_id:
    params["title"] = new_id
hass.config_entries.async_update_entry(entry, **params)
```